### PR TITLE
Update remi & use php 8 on CentOS

### PR DESCRIPTION
**Pull request by markd69**
Update Remi to 9 and use PHP 8.0

Also noted that CentOS 8 is EOL.

### CODE REVIEW

File: community/installation-guides/panel/centos8.md [hunk 1]
```diff
@@ -34,8 +34,8 @@ We recommend the remi repo to get the latest php packages.
 ```bash
 ## Install Repos
 dnf install epel-release
-dnf install https://rpms.remirepo.net/enterprise/remi-release-8.rpm
-dnf module enable php:remi-7.4
+dnf install https://rpms.remirepo.net/enterprise/remi-release-9.rpm
```
Comment (Software-Noob) [inline community/installation-guides/panel/centos8.md:37]: This should be 8 and not 9

**Review by Software-Noob — CHANGES_REQUESTED** [review]

**Comment by Software-Noob** [comment]
Can you please update all the other sections that specify PHP 7.4 to use 8.0 instead for consistency? There's the title in centos8.md and comment in centos7.md that still mentions PHP 7.4

**Comment by markd69** [comment]
> Can you please update all the other sections that specify PHP 7.4 to use 8.0 instead for consistency? There's the title in centos8.md and comment in centos7.md that still mentions PHP 7.4

Sorry about that, done.

**Review by Software-Noob — APPROVED** [review]
LGTM, thank you for contributing :+1:

**Pull request merged**